### PR TITLE
OFMCC-1829 - Permissions for Reporting

### PR DESCRIPTION
**Pull request by weskubo-cgi**
Secured reporting permissions.
Added stub code for delete.

### CODE REVIEW

File: frontend/src/views/reports/SurveyView.vue [hunk 1]
```diff
@@ -83,7 +85,7 @@ export default {
   computed: {
```
Comment (jenbeckett) [inline frontend/src/views/reports/SurveyView.vue:85]: this is older code, but Line 81 : var deletedTableResponses is unused

**Review by jenbeckett — CHANGES_REQUESTED** [review]

File: frontend/src/views/reports/SurveyView.vue (hunk 1 above)
Comment (weskubo-cgi) [reply frontend/src/views/reports/SurveyView.vue:85]: Blame @vietle-cgi :).

**Review by vietle-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/components/reports/PendingReportsTab.vue [hunk 2]
```diff
@@ -149,6 +148,19 @@ export default {
     facilityFilterChanged(newVal) {
       this.facilityNameFilter = newVal
     },
+
+    showOpen(surveyResponse) {
```
[the remark this answers is not in the document]
Comment (weskubo-cgi) [reply frontend/src/components/reports/PendingReportsTab.vue:152]: No reason. Just how I refactored it. Updated.

File: frontend/src/views/reports/SurveyView.vue (hunk 1 above)
Comment (vietle-cgi) [reply frontend/src/views/reports/SurveyView.vue:84]: I can remove this var when I work on MR next sprint

**Review by jenbeckett — APPROVED** [review]

**Pull request merged**